compute_tools/src/compute.rs

```diff
@@ -1024,6 +1075,7 @@ impl ComputeNode {
                 CreateAndAlterRoles,
                 RenameAndDeleteDatabases,
                 CreateAndAlterDatabases,
+                CreateSchemaNeon,
```

Comment:
Personal opinion: CreateSchemaNeon seems to be a very fundamental thing, so I'd just move to the very top, e.g. right after `CreateSuperUser`, but up to you